DESCRIPTION

```diff
@@ -28,3 +28,4 @@ Suggests:
 VignetteBuilder: knitr
 LazyLoad: yes
 LazyData: yes
+RoxygenNote: 7.3.1
```

Comment:
@dan-knight I think I've brought up using Roxugen in the past, do we want to use this for the rest of the documentation as well? I think if we start using it, we should standardize it.

Reply:
Yeah, we're not using Roxygen in this package. It doesn't make sense to sneak it into this change.

Reply:
From our conversation, this came from running `devtools::document()` so I could use these functions locally. I don't intend to introduce Roxygen, so this can be removed.